L/LLVM_jll/Deps.toml

```diff
@@ -1,3 +1,3 @@
-[9]
+[6]
```

Comment:
I think we should have a section for each major version, all the sections will be the same

Reply:
It looks like the dependencies in the Deps.toml file are increasingly added: https://github.com/JuliaRegistries/General/blob/master/D/DiffEqBase/Deps.toml